config/crd/bases/infrastructure.cluster.x-k8s.io_openstackmachinetemplates.yaml

```diff
@@ -15,6 +15,8 @@ spec:
     kind: OpenStackMachineTemplate
     listKind: OpenStackMachineTemplateList
     plural: openstackmachinetemplates
+    shortNames:
```

Comment:
curious why we have 3 changes but only one template update here?

Reply:
I was also wondering. 
I did run ```make generate*``, but no other template was created.